website/content/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s.mdx

```diff
@@ -0,0 +1,71 @@
+---
+layout: docs
+page_title: Use JWTs to verify requests to API gateways on Kubernetes
+description: Learn how to use JSON web tokens (JWT) to verify requests from external clients to listeners on an API gateway on Kubernetes-orchestrated networks.
+---
+
+# Use JWTs to verify requests to API gateways on Kubernetes
+
+This topic describes how to use JSON web tokens (JWT) to verify requests to API gateways deployed to Kubernetes-orchestrated containers. If your API gateway is deployed to virtual machines, refer to [Use JWTs to verify requests to API gateways on VMs](/consu/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-vms).
+
+## Overview
+ 
+You can configure API gateways to use JWTs to verify incoming requests so that you can stop unverified traffic at the gateway. You can configure JWT verification at different levels:
+
+- Listener defaults: Define basic defaults that apply to all routes attached to a listener.
+- HTTP route-specific settings: You can define JWT authentication settings for specific HTTP routes. Route-specific JWT settings override default configurations. 
+- Listener overrides: Define override settings that take precedence over default and route-specific configurations. This enables you to set enforceable policies for listeners.
+
+
+Complete the following steps to use JWTs to verify requests:
```

Comment:
we also need to include the jwt provider, like is defined here https://developer.hashicorp.com/consul/docs/connect/intentions/jwt-authorization#create-a-jwt-provider-configuration-entry, the jwt providers in the gateway policy and route auth filter need to reference an existing jwt provider by name

Reply:
(this also applies to VM's)

Reply:
This supposed to be an overview section. I think what you're describing is a detail that we cover at line 43.

Reply:
can you link to where it's mentioned that the provider stanza has to reference a JWT provider by name? I think I'm missing where that's stated